torch/testing/_internal/distributed/rpc/rpc_test.py

```diff
@@ -2692,6 +2692,12 @@ def test_rref_context_debug_info(self):
         # clear states for check 2
         rpc.rpc_sync(worker_name(dst_rank), clear_global_rref)
 
+        # Wait for owner rref to be cleared.
+        while int(info["num_owner_rrefs"]) != 0:
```

Comment:
Is there any chance for the delete operation to fail in this particular test? In such case this loop would basically run forever and block the test process. Would it make sense to have a timeout threshold?

Reply:
The overall test has a timeout, so this loop won't block forever and the test would timeout. We also dump python traces as part of the timeout, so if this happens the python trace would indicate where we are blocked.